docs/creating_an_api_server.md

```diff
@@ -2,7 +2,7 @@
 
 **Note:** This document explains how to manually create the files generated
 by `apiserver-boot`.  It is recommended to automatically create these files
-instead.  See [getting started](https://sigs.k8s.io/apiserver-builder-alpha/blob/master/docs/getting_started.md)
+instead.  See[api_building_overview](https://sigs.k8s.io/apiserver-builder-alpha/blob/master/docs/concepts/api_building_overview.md)
```

Comment:
This link doesn't seem to work when I click on it.  Try https://github.com/kubernetes-sigs/apiserver-builder-alpha/blob/master/docs/concepts/api_building_overview.md